modcc/symdiff.cpp

```diff
@@ -266,7 +267,13 @@ class SymPDiffVisitor: public Visitor, public error_stack {
 // from the given one with constant simplifications.
 
 long double expr_value(Expression* e) {
-    return e && e->is_number()? e->is_number()->value(): NAN;
+#ifdef __FAST_MATH__
```

Comment:
I think we should just avoid fast math, or worst case it is something that should be turned on in files where desired.